aten/src/ATen/native/native_functions.yaml

```diff
@@ -1363,6 +1363,18 @@
     CPU: batch_norm_update_stats_cpu
     CUDA: batch_norm_update_stats_cuda
 
+- func: native_layer_norm(Tensor input, Tensor? weight, Tensor? bias, int[] normalized_shape, float eps) -> (Tensor, Tensor, Tensor)
```

Comment:
can you give these outputs names, e.g. `(Tensor output, Tensor mean, Tensor rstd)` or similar?

Reply:
Done.

Reply:
It seems this change will break the yaml test. I got "only whitelisted operators are allowed to have named return type, got native_layer_norm". Just removed the return tensor names.